template-processors/base/bin/wrapper.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# shellcheck source=$HOME/envs.sh
+# shellcheck disable=SC1091
```

Comment:
AFAIU, above 2 lines should not be used together. Instead, please remove l.3, and move l.4 above l.28 - if this will work.